bin/json-schema-to-typescript.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var program = require('commander');
+
+var pkg = require('../package.json');
+
+var lib = require('../dist/index.js');
+var compileFromFile = lib.compileFromFile;
+
+program
+  .version(pkg.version)
+  .command('json-schema-to-typescript <in> [out]', 'convert a JSON-schema <in> file and output a TypeScript [out] file, or stdout if [out] file is not specified')
+  .action(function (inFile, outFile) {
+    outFile = (typeof outFile === 'string' ? outFile : null);
+
+    return compileFromFile(inFile).then(function (output) {
+      if (!outFile) {
+        console.log(output);
```

Comment:
This should probably write to process.stdout to avoid inserting extra formatting characters in the output.